docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+services:
+  postgres:
+    image: postgres:9.5.4
+    ports:
+      - "5433:5432"
```

Comment:
port in compose file doesn't match with https://github.com/khaiql/dbcleaner/pull/2/files#diff-8fe505853e03f71a9e4908cf26e6da2bR13